includes/llms.template.functions.php

```diff
@@ -385,6 +385,32 @@ function lifterlms_template_single_parent_course() {
 	}
 }
 
+/**
+ * Favorite Lesson Template Include.
+ *
+ * @since [version]
+ *
+ * @param int    $object_id   WP Post ID of the Lesson, Section, Track, or Course.
+ * @param string $object_type Object type [lesson|section|course|track].
+ * @return void
+ */
+if ( ! function_exists( 'lifterlms_template_favorite' ) ) {
```

Comment:
`llms_` is the correct prefix.
https://github.com/gocodebox/lifterlms/blob/trunk/docs/coding-standards.md#functions--variables

Reply:
fixed.